install/kubernetes/helm/istio/templates/configmap.yaml

```diff
@@ -85,6 +85,24 @@ data:
     outboundTrafficPolicy:
       mode: {{ .Values.global.outboundTrafficPolicy.mode }}
 
+    {{- if .Values.global.configRootNamespace }}
+    # The namespace to treat as the administrative root namespace for istio
+    # configuration. Set this field to a dedicated namespace if you want to
+    # all sidecars to be able to communicate with services in their
+    # namespace alone. This dedicated namespace should have a default
+    # Sidecar config
+    rootNamespace: {{ .Values.global.configRootNamespace }}
+    {{- end }}
+
+    {{- if .Values.global.defaultConfigExportSetting }}
+    defaultServiceExportTo:
```

Comment:
Same, plus: is the value a single value ? Didn't we have a list of namespaces ?

Reply:
With regard to the single value, its just the current limitation for 1.1. The list of namespaces will require more indices in Pilot. Right now, the . and * map to private and public neatly. which means code changes are much less

Reply:
Code limitation can be fixed - values.yaml churn is a different story, we should treat it as an API. 
Can we have a list here ( with docs saying that in 1.1 it can be only one item ) ?

Reply:
done